### DISCUSSION TITLE
Adding and Customize Page

### DISCUSSION THREAD

**Discussion by duygugunes** [inventree/InvenTree#4321] [General]
Hi,

I want to adding new form and also it would be helpfull to customize the stock screen, like adding new combobox and textbox to stock screen. Are two function(adding scrren and customizing screen) possible with inventree?
In addition that, some validation and multiple approval process is neccessary for my project. How can i do this?

**Comment by matmair** [comment]
I'd suggest looking into plugins https://docs.inventree.org/en/latest/extend/how_to_plugin/
